workspaces/mujoco_workspace.py

```diff
@@ -77,16 +81,30 @@ def train(self):
                 self.save_snapshot()
 
             if done:
+                ret = rew[:]
+                ret[-1] = np.mean(rew[-100:])*99
+
+                for i in reversed(range(len(rew) - 1)):
+                    ret[i] = rew[i] + 0.99 * ret[i+1]
+                xs = list(range(len(ret)))
+                values = np.array(val_seq)
+                print("Episode: {}, total numsteps: {}, return: {}".format(self._train_episode, self._train_step, round(info["episode"]["r"][0], 2)))
+
                 self._train_episode += 1
-                print("Episode: {}, total numsteps: {}, return: {}".format(self._train_episode, self._train_step, round(info["episode"]["r"], 2)))
                 if self.cfg.wandb_log:
+                    plt.plot(xs, ret, xs, values)
+                    wandb.log({"ret_val_diff" : plt})
                     episode_metrics = dict()
-                    episode_metrics['episodic_length'] = info["episode"]["l"]
-                    episode_metrics['episodic_return'] = info["episode"]["r"]
-                    episode_metrics['steps_per_second'] = info["episode"]["l"]/(time.time() - episode_start_time)
+                    episode_metrics['episodic_length'] = info["episode"]["l"][0]
```

Comment:
Why did those [0] appear? They feel unnecessary

Reply:
My version of WANDB doesn't seem to take the list as input, so I have to index it somehow.